### ISSUE TITLE
Support multi-line for yaml within the same row

### ISSUE THREAD

**Issue by pascalwhoop** [timvink/mkdocs-table-reader-plugin#47]
<img width="411" alt="image" src="https://user-images.githubusercontent.com/2284951/229810429-a336fbfd-1cba-47d8-bdd5-fc2fd3a10ac0.png">
<img width="179" alt="image" src="https://user-images.githubusercontent.com/2284951/229810462-9c879712-7e4f-4248-a8f0-c8bc3e4725d5.png">

When using multi-line strings it converts it to multiple lines in markdown. Any chance to get this to render as a single cell with multiple lines (as in `<br/>`) instead?

**Comment by timvink** [comment]
Hi Pascal,

I need a little more context. Perhaps you could make a small reproducable example?

I'm guessing you're using `read_yaml` ? That in turn uses `yaml.safe_load()`  to load to a python dictionary. That is then turned into a pandas dataframe using `pd.json_normalize()`, and then converted to markdown.

yaml is a tricky 'language' -- it seems to me that the definition column actually has a list of 3 items instead of a multi-line string.  

It's highly unlikely I can do something in this package to change the behaviour

**Comment by timvink** [comment]
Just came across this page on YAML and multi-line strings and thought of this issue, might be helpful:  https://yaml-multiline.info/

**Comment by timvink** [comment]
TLDR; `mkdocs`  does not support this!

--- 

Ok, I played around with it a bit:

```yaml
- term: prompt
  definition: |
   A
   B
   C
- term: else
  definition: cool
```

```python
with open(args[0], "r") as f:
    j = list(yaml.safe_load(f))
df = pd.json_normalize(j
``` 

This gives what we expect:

![image](https://github.com/timvink/mkdocs-table-reader-plugin/assets/5570380/51d4c694-0f6a-45f8-9408-b9be0c74cdb6)

It's this part that is the 'problem':

![image](https://github.com/timvink/mkdocs-table-reader-plugin/assets/5570380/f7ae3e7d-4210-479c-9cce-836031cb5995)

That part is taken care of by the `tabulate` poackage. There we use the tablefmt `pipe` because it's compatible with mkdocs. And that format does _not_ support multiline strings.

Which in turns brings me to the mkdocs website (https://www.mkdocs.org/user-guide/writing-your-docs/):

![image](https://github.com/timvink/mkdocs-table-reader-plugin/assets/5570380/24875254-e874-483d-8c33-e1141361b334)

:(
